static/src/javascripts-legacy/projects/common/modules/crosswords/helpers.js

```diff
@@ -291,7 +291,7 @@ define([
                 map[d.key] = {};
             }
 
-            map[d.key][d.direction] = d.separator;
+            map[d.key] = d;
```

Comment:
This remapping complicated things in the grid module and I couldn't find it being used elsewhere.